src/routes/Apps/App/Testing.css

```diff
@@ -0,0 +1,79 @@
+.cl-testing-header {
```

Comment:
I find it hard to believe that these are all top level classes. I would assume most of them are children, would change them to follow the rest of the BEM classes. http://getbem.com/naming/